docusaurus.config.js

```diff
@@ -7,6 +7,35 @@ if (!process.env.CI) {
     : 'fintan-contents/mobile-app-crib-notes';
 }
 
+const injector = (options) => {
+  const keys = Object.keys(options);
+  const placeHolders = keys.map((key) => new RegExp('{@inject: ?' + key + '}'));
```

Comment:
スペースの数はこだわらなくていいんじゃないかな、って。
```suggestion
  const placeHolders = keys.map((key) => new RegExp('{@inject: *' + key + '}'));
```

Reply:
1e7b664fb9a5d85044e4620262ca0b30ca9d4c97
出来た...